vector_search/encoders/dummy.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+from vector_search.encoders.base import BaseEncoder
+
+
+class DummyEmbedEncoder(BaseEncoder):
```

Comment:
This only seems to be used for unit tests, maybe it could could go in a `vector_search/conftest.py` file instead?